Lib/pathlib/__init__.py

```diff
@@ -90,7 +90,8 @@ def __init__(self, *args):
                         "object where __fspath__ returns a str, "
                         f"not {type(path).__name__!r}")
                 paths.append(path)
-        super().__init__(*paths)
+        self._raw_paths = paths
```

Comment:
Add a comment so somebody doesn't create a PR in a year's time "simplifying" this code? ;)

```suggestion
        # Avoid calling super().__init__, as an optimisation
        self._raw_paths = paths
```